sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateMutableProjection.scala

```diff
@@ -60,38 +60,32 @@ object GenerateMutableProjection extends CodeGenerator[Seq[Expression], MutableP
     val projectionCodes = exprVals.zip(index).map {
```

Comment:
now it's better to explicit write down the type, e.g. `val projectionCodes: Seq[...]` and explain the meaning